client/src/components/Header.jsx

```diff
@@ -36,32 +37,34 @@ export default function Header({
   const navigateTo = (route) => {
     switch (route) {
       case 'My Recipes':
-        navigate(`/user/${user.name}/recipes`);
+        navigate(`/my-recipes`);
```

Comment:
Thinking ahead a bit, if you want users to access someone else's recipes, would it make more sense for consistency to use a version of `/username/recipes` endpoint?

Reply:
I think if someone wants to see someone else's recipes it could be done via parameters to a /recipes or /all-recipes navigation.

It's unless likely for someone to search for recipes from a specific user than something like "all recipes with beef in it" or "give me some random recipes you think i'd like"

Reply:
You can already view others' recipes in the home page. The home page is meant to display the collection of recipes in database, where anybody can browse and search. The point of My Recipes page is specifically for authenticated user to view own recipes and/or modify them. So that's why I removed the username from the path. This prevents a different user from possibly accessing someone's recipes and modify them in the frontend. Of course, I'll need to add something to prevent this in the backend.

Reply:
done